main/inc/lib/tracking.lib.php

```diff
@@ -1898,104 +1944,195 @@ public static function getLpStats(
+                                                        </td>';
+                                                    } else {
+                                                        $output .= '<td>
+                                                        <a href="../exercise/exercise_show.php?origin='.$origin.'&id='.$my_exe_id.'&cidReq='.$courseCode.'" target="_parent">
+                                                        <img src="'.Display::returnIconPath('quiz.png').'" alt="'.get_lang('ShowAttempt').'" title="'.get_lang('ShowAttempt').'">
+                                                        </a></td>';
+                                                    }
                                                 } else {
-                                                    $output .= '<td>
-                                                            <a href="../exercise/exercise_show.php?origin='.$origin.'&id='.$my_exe_id.'&cidReq='.$courseCode.'" target="_parent">
-                                                            <img src="'.Display::returnIconPath('quiz.png').'" alt="'.get_lang('ShowAttempt').'" title="'.get_lang('ShowAttempt').'">
-                                                            </a></td>';
+                                                    if (!$is_allowed_to_edit && $result_disabled_ext_all) {
+                                                        $output .= '<td>
+                                                            <img src="'.Display::returnIconPath('quiz_na.gif').'" alt="'.get_lang('ShowAndQualifyAttempt').'" title="'.get_lang('ShowAndQualifyAttempt').'"></td>';
+                                                    } else {
+                                                        $output .= '<td>
+                                                                <a href="../exercise/exercise_show.php?cidReq='.$courseCode.'&origin=correct_exercise_in_lp&id='.$my_exe_id.'" target="_parent">
+                                                                <img src="'.Display::returnIconPath('quiz.gif').'" alt="'.get_lang('ShowAndQualifyAttempt').'" title="'.get_lang('ShowAndQualifyAttempt').'"></a></td>';
+                                                    }
                                                 }
+                                            }
+                                            $output .= '</tr>';
+                                            $n++;
+                                        }
+                                    }
+                                    $output .= '<tr><td colspan="12">&nbsp;</td></tr>';
+                                }
+                                if (
+                                    'h5p' === $row['item_type']
+                                    && 'true' === api_get_plugin_setting('h5pimport', 'tool_enable')
```

Comment:
`api_get_plugin_setting('h5pimport', 'tool_enable')` has already been checked before, so we could store the result in a local variable and re-use the variable (for increased efficiency).